README.md

```diff
@@ -77,4 +77,4 @@ may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
 Unless required by applicable law or agreed to in writing, software distributed
```

Comment:
The software constructs all or part of an SQL command using externally-influenced input from an upstream component, but it does not neutralize or incorrectly neutralizes special elements that could modify the intended SQL command when it is sent to a downstream component.

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix | **Sema Tags:** Not secure